app/views/planning_application/site_notices/new.html.erb

```diff
@@ -0,0 +1,94 @@
+<% content_for :page_title do %>
+  Site notice - <%= t('page_title') %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", root_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+<% add_parent_breadcrumb_link "Consultations", planning_application_consultations_path(@planning_application) %>
+<% content_for :title, "Site notice" %>
+
+<%= render(
+  partial: "shared/proposal_header",
+  locals: { heading: "Send site notice" }
+) %>
+
+<% if @planning_application.site_notices.any? %>
+  <hr>
+  <div class="proposal-details-sub-heading govuk-!-margin-top-3 govuk-!-margin-bottom-3">
+    <p class="govuk-body govuk-!-margin-top-1">
+      <%= @planning_application.last_site_notice_audit.audit_comment %>
+    </p>
+    <div class="consultation-letter-status">
+      <div>
+        <%= render(StatusTags::BaseComponent.new(status: :complete)) %>
+      </div>
+      <p class="govuk-body-s govuk-!-margin-top-1 govuk-!-margin-bottom-0">
+        on <%= @planning_application.last_site_notice_audit.created_at.to_fs(:day_month_year_slashes) %>
+      </p>
+    </div>
+  </div>
+  <hr>
+<% end %>
+
+<div data-controller="pdf">
+  <%= form_with(
+    model: [@planning_application, @site_notice],
+    builder: GOVUKDesignSystemFormBuilder::FormBuilder,
+    class: "govuk-!-margin-top-5",
+    method: :post
+  ) do |form| %>
+    <div data-controller="show-hide-form">
+      <div class="display-none">
+        <div id="site-notice-content">
+          <div style="width: 430px;">
+            <%= @site_notice.preview_content(@planning_application.planning_application).html_safe %>
```

Comment:
Ha what is `@planning_application.planning_application`

Reply:
When we set @planning_applcation in views, it's usually the planning application view, which seems to only load some of the attributes. If you want the full planning application you have to do @planning_application.planning_application 😂

Reply:
Ahh is it because we are initialising like 
```
PlanningApplicationPresenter.new(view_context, planning_application)
```